src/plugins/saved_objects_management/public/management_section/object_view/components/inspect.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { XJsonLang } from '@kbn/monaco';
+import { omit } from 'lodash';
+import { EuiButtonEmpty, EuiCopy, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { CodeEditor } from '../../../../../kibana_react/public';
+import { SavedObjectWithMetadata } from '../../../../common';
+
+export interface InspectProps {
+  object: SavedObjectWithMetadata<any>;
+}
+const codeEditorAriaLabel = (title: string) =>
+  i18n.translate('savedObjectsManagement.view.inspectCodeEditorAriaLabel', {
+    defaultMessage: 'inspect { title }',
+    values: {
+      title,
+    },
+  });
+const copyToClipboardLabel = i18n.translate('savedObjectsManagement.view.copyToClipboardLabel', {
+  defaultMessage: 'Copy to clipboard',
+});
+export const Inspect = ({ object }: InspectProps) => {
+  const title = object.meta.title || 'saved object';
+  const objectAsJsonString = JSON.stringify(omit(object, 'meta'), null, 2);
```

Comment:
NIT: `useMemo` here to avoid stringifying the object on each render

```ts
import React, { useMemo } from 'react';


const objectAsJsonString = useMemo(() => {
   return JSON.stringify(omit(object, 'meta'), null, 2);
}, [object]);
```